### ISSUE TITLE
Laplace RealNVP

### ISSUE THREAD

**Issue by KeyKy** [Jeff-sjtu/res-loglikelihood-regression#18]
I only find the implementation of MultivariateNormal of RealNVP. Could you open source your Laplace of RealNVP？I do not find MultivariateLaplace in pytorch.

**Comment by jeffffffli** [comment]
Hi @KeyKy, we only use the multivariate normal RealNVP. The Laplace distribution mentioned in the paper is about the Q distribution in the loss function, not the flow model.
